third_party/terraform/resources/resource_compute_instance_template.go

```diff
@@ -563,6 +563,12 @@ func buildDisks(d *schema.ResourceData, config *Config) ([]*computeBeta.Attached
 
 		if v, ok := d.GetOk(prefix + ".source"); ok {
 			disk.Source = v.(string)
+			conflicts := []string{"disk_size_gb", "disk_name", "disk_type", "source_image", "labels"}
```

Comment:
`labels` doesn't exist in schema yet but it will be added in https://github.com/terraform-providers/terraform-provider-google/pull/3766